src/BeadingStrategy/DistributedBeadingStrategy.h

```diff
@@ -17,22 +17,26 @@ namespace cura
  */
 class DistributedBeadingStrategy : public BeadingStrategy
 {
-private:
+protected:
```

Comment:
How come this is protected? I don't see any other classes inheritting from the DistributedBeadingStrategy.

Reply:
So in that case it also doesn't matter. It's maybe a matter of personal preference, but i was taught to just make things protected if it make sense that if you ever do make a child class that it could do something with it.

Reply:
Agreed.